ios/engine/KMEI/KeymanEngine/Classes/Manager.swift

```diff
@@ -342,27 +342,9 @@ public class Manager: NSObject, UIGestureRecognizerDelegate {
 
   /// Adds a new keyboard to the list in the keyboard picker if it doesn't already exist.
   /// The keyboard must be downloaded (see `downloadKeyboard()`) or preloaded (see `preloadLanguageFile()`)
+  @available(*, deprecated, message: "Deprecated in favor of ResourceFileManager.install(resourceWithID:from:)")
```

Comment:
These deprecated methods (+ l. 392) can be listed in 
https://github.com/keymanapp/keyman/wiki/Keyman-13.0-to-14.0-Migration and help.keyman.com

Will this affect KMSample and FV apps too?

Reply:
Yes, it will.  That said, I've taken care to keep them just as functional as before.

They're still supported, though their existence does make things messy... hence the deprecation warning.

Reply:
It does seem that I'll need some extra work with an extra PR to fully cover these cases, now that I'm inspecting things again.  The `preloadFiles` -> `addKeyboard` combo won't produce the desired `kmp.json` at present.  I'll address this in another PR.

Reply:
Since ResourceFileManager.install() will be added to the API, can you add a description in ResourceFileManager.install()? (l.220)

Reply:
Added in #3247.